### ISSUE TITLE
Drop magic number 16707566 (0xFeeFee)

### ISSUE THREAD

**Issue by amai2012** [OpenCppCoverage/OpenCppCoverage#97]
**Describe the bug**
OpenCppCoverage spits out coverage for line 16707566 - which does not exist in the given file since it seems to a magic number (see https://blogs.msdn.microsoft.com/jmstall/2005/06/19/line-hidden-and-0xfeefee-sequence-points/) 
I suggest to remove this from the output since the result should be coverage which can be mapped to the source code.

**To Reproduce**
Run opencppcoverage on my closed source project

**Expected behavior**
Remove that entry fom the results
` <line number="16707566" hits="0"/>`

**Desktop (please complete the following information):**
- Version of OpenCppCoverage: 0.9.8.0
- Architecture (x86/ x64):  x64
- Windows version:  any

**Additional context**

**Comment by OpenCppCoverage** [comment]
Hello,

Thank you for reporting the issue.
Old version correctly handle this line [here](https://github.com/OpenCppCoverage/OpenCppCoverage/blob/release-0.9.5.3/CppCoverage/FileDebugInformation.cpp#L68), but now it handles only [0x00f00f00](https://github.com/OpenCppCoverage/OpenCppCoverage/blob/master/CppCoverage/DebugInformationEnumerator.cpp#L289). 

OpenCppCoverage

**Comment by amai2012** [comment]
I've found another reference: https://github.com/dotnet/corefx/blob/master/src/System.Reflection.Metadata/specs/PortablePdb-Metadata.md#MethodDebugInformationTable

Is there public documentation about `0x00f00f00`  available?

**Comment by OpenCppCoverage** [comment]
If there is a documentation, I do know it.

**Comment by OpenCppCoverage** [comment]
Hi,

The following code generates a line number 16707566 (`0xFeeFee`).

```c++
int main(int argc, char* argv[])
{
	try
	{	
	}
	catch (...)
	{
	}
	return 0;
}
```

When the file exists, the line number 16707566 is filtered by https://github.com/OpenCppCoverage/OpenCppCoverage/blob/fcab13f10b198f3b7ef1e2aef2f9a38c3ea50352/FileFilter/LineFilter.cpp#L60-L65

[This commit](https://github.com/OpenCppCoverage/OpenCppCoverage/commit/811c323a93c8593f27c023c0c076b1c4cbe31880) filters both `0x00f00f00` and `0x00feefee` to handle the case when the file does not exist.

OpenCppCoverage

**Comment by amai2012** [comment]
Since the link from my first is gone meanwhile I add the bottom line here for reference:
```Debuggers + compilers have a common agreement that if a line maps 0xFeeFee, then the debugger will skip over it.  So the ‘#line hidden’ directive tells the C# compiler to map all following lines to a 0xFeeFee line number, instead of the real line number.```

**Comment by OpenCppCoverage** [comment]
Fixed in [OpenCppCoverage 0.9.9.0](https://github.com/OpenCppCoverage/OpenCppCoverage/releases/tag/release-0.9.9.0).
